topics/ecology/tutorials/Obitools-metabarcoding/tutorial.md

```diff
@@ -0,0 +1,537 @@
+>    {: .comment}
+>
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> 1. How many sequences are not assigned?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. 1391
+> >
+> {: .solution}
+>
+{: .question}
+
+## Dereplicate reads into uniq sequences with **obiuniq**
+
+> ### {% icon hands_on %} Hands-on: Groups together sequence records
+>
+> 1. {% tool [obiuniq](toolshed.g2.bx.psu.edu/repos/iuc/obi_uniq/obi_uniq/1.2.13) %} with the following parameters:
+>    - *"Input sequences file"*: `Trimmed and annotated file by NGSfilter`
+>    
+>    - *"Attribute to merge"*: `sample`
+>    
+>    - *"Use specific option"*: `merge`
+>
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > The same DNA molecule can be sequenced several times. In order to reduce both file size and computations time, and to get easier interpretable results, it is convenient to work with unique sequences instead of reads. To dereplicate such reads into unique sequences, we use the obiuniq command.
+>    > Definition: Dereplicate reads into unique sequences
+>    > * compare all the reads in a data set to each other
+>    > * group strictly identical reads together
+>    > * output the sequence for each group and its count in the original dataset (in this way, all duplicated reads are removed)
+>    > *Definition adapted from Seguritan and Rohwer (2001)*
+>    {: .comment}
+>
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> 1. How many sequences you had and how many you finally obtain?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. From 43 326 to 3 962
+> >
+> {: .solution}
+>
+{: .question}
+
+
+## Limit number of informations with **obiannotate**
+
+> ### {% icon hands_on %} Hands-on: Adds/Edits sequence record annotations
+>
+> 1. {% tool [obiannotate](toolshed.g2.bx.psu.edu/repos/iuc/obi_annotate/obi_annotate/1.2.13) %} with the following parameters:
+>    - In *"Keep only attribute with key"*:
+>        - *"key"*: `count`
+>        - *"if you want to specify a second key"*: `merged_sample`
+
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > obiannotate is the command that allows adding/modifying/removing annotation attributes attached to sequence records.
+>    > Once such attributes are added, they can be used by the other OBITools commands for filtering purposes or for statistics computing.
+>    > 
+>    > Here, the goal is to keep only `count` and `merged_sample` key=value attributes! 
+>    {: .comment}
```

Comment:
Also I would move the content of this block at the beginning.

Reply:
done! THANK YOU !!!